scripts/delivery.json

```diff
@@ -48,6 +48,14 @@
         { "name": "mobile_number", "type": "string", "required": false },
         { "name": "email_id", "type": "string", "required": false }
       ]
+    },
+    
+    "delivery_form" : {
```

Comment:
This should look similar to delivery model with all the fields except `id` and `estimated_delivery_date`